shuup/xtheme/locale/fi_FI/LC_MESSAGES/django.po

```diff
@@ -320,18 +352,21 @@ msgstr "Aktivoi teema nähdäksesi ohjeistuksen"
 msgid "No guide"
 msgstr "Ei ohjeistusta"
 
-msgid "Currently active"
-msgstr "Aktiivinen"
-
 msgid "Activate"
 msgstr "Aktivoi"
 
+msgid "Click here to configure the theme"
+msgstr "Klikkaa tähän asentaaksesi teeman."
+
 msgid "Configure"
 msgstr "Muokkaa"
 
 msgid "Theme Name"
 msgstr "Teeman nimi"
 
+msgid "Currently active"
+msgstr "Tällä hetkellä aktiivinen"
```

Comment:
`Aktiivinen tällä hetkellä` unless the full thing is gonna be like "currently active: xxxx" then the sentence you wrote is ok.